loadbalancer/zk/logwrapper.go

```diff
@@ -0,0 +1,21 @@
+package zk
+
+import (
+	"fmt"
+	"github.com/go-kit/kit/log"
+	"github.com/samuel/go-zookeeper/zk"
+)
+
+type wrapLogger struct {
+	log.Logger
+}
```

Comment:
Is this only used for the Printf convenience function? If so, please remove it, and use `logger.Log("msg", "...")` directly.